packages/generator/src/main/ts/openapi.ts

```diff
@@ -0,0 +1,109 @@
+import { OpenAPIV3 } from 'openapi-types'
+
+import { getActions, getLinks } from './parser'
+import { TParameter, TParsedPage } from './types'
+import { normalizeName } from './utils'
+
+export function formatType(type: string) {
+  const typeMap = {
+    String: 'string',
+    Integer: 'number',
+    Unknown: 'string',
+    Boolean: 'boolean',
+  }
+
+  const arrayPrefix =
+    type[0] === '[' && type[type.length - 1] === ']' ? '[]' : ''
+  if (arrayPrefix) {
+    type = type.slice(1, -1)
+  }
```

Comment:
This should a single condition.

 if (type[0] === ...) {
   type = type.slice(1, -1)
}